linalg/sparsemat.cpp

```diff
@@ -50,6 +58,10 @@ SparseMatrix::SparseMatrix(int nrows, int ncols)
 #ifdef MFEM_USE_MEMALLOC
    NodesMem = new RowNodeAlloc;
 #endif
+
+#ifdef MFEM_USE_CUDA
```

Comment:
Since you have the ifdef in the function you don't need it in these calls.  Anyone else want to comment on this belt and suspenders style?